README.md

```diff
@@ -1,13 +1,131 @@
++ password (string) - Password for user, length must be greater than 6
++ confirmPassword (string) - Confirm password for validation
+
+#### Signs in a user - ```POST /api/v1/users/signin```
++ email (string) - User's email
++ password (string) - User's password
+
+#### Edits a user - ```POST /api/v1/users/edit```
+ Requires authentication
++ firstName (string) - User's first name
++ lastName (string) - User's last name
++ email (string) - User's email
++ profilePic (string) - URL to user's profile picture
++ about (string) - User's about
+
+#### Get user by id - ```GET /api/v1/user/{user_id}```
+
+#### Delete a user - ```DELETE /api/v1/user/{user_id}```
+ Requires authentication
+
+#### Gets recipes created by user - ```GET /api/v1/users/recipes```
+ Requires authentication
+
+#### Adds favourite recipe for a user - ```POST /api/v1/users/recipes/{recipe_id}/favourites```
+ Requires authentication
+
+#### Deletes favourite recipe for a user - ```DELETE /api/v1/users/recipes/{recipe_id}/favourites```
+ Requires authentication
+
+#### Gets all favourite recipes for a user - ```GET /api/v1/users/recipes/favourites```
+ Requires authentication
+
+#### Adds a recipe - ```POST /api/v1/recipes```
+ Requires authentication
++ name (string) - Name of the recipe
++ description (string) - Description of the recipe
++ ingredients (string) - Ingredients for the recipe, it should be a comma-separated string of ingredients, e.g 'pepper,salt,sauce'
+
+#### Gets all recipes - ```GET /api/v1/recipes```
+
+#### Gets all recipes and sorts them by votes - ```GET /api/v1/recipes?sort=upvotes&order=asc```
+
+#### Searches for a recipe with query - ```GET /api/v1/recipes?query=search```
+
+#### Paginates recipe - ```GET /api/v1/recipes?from=0&to=5```
+
+#### Gets recipe by id - ```GET /api/v1/recipes/{recipe_id}```
+ Get recipe with the specified id in the url
+
+#### Edits recipe - ```PUT /api/v1/recipes/{recipe_id}```
+ Requires authentication. User can only edit a recipe they created.
+ Edits recipe with the specified id in the url. Any one of the parameters below can be passed
++ name (string) - Name of the recipe
++ description (string) - Description of the recipe
++ ingredients (string) - Ingredients for the recipe, it should be a comma-separated string of ingredients, e.g 'pepper,salt,sauce'
+
+#### Deletes recipe - ```DELETE /api/v1/recipes/{recipe_id}```
+ Requires authentication. User can only delete a recipe they created
+
+#### Add review for a recipe - ```POST /api/v1/recipes/{recipe_id}/reviews```
+ Requires authentication
++ review (string) - Review
+
+#### Edit review - ```PUT /api/v1/recipes/reviews/{review_id}```
+ Requires authentication
+
+#### Delete review - ```DELETE /api/v1/recipes/reviews/{review_id}```
+ Requires authentication
+
+#### Upvote a recipe - ```POST /api/v1/recipes/upvote/{recipe_id}```
+ Requires authentication
+
+#### Downvote a recipe - ```POST /api/v1/recipes/downvote/{recipe_id}```
+ Requires authentication
```

Comment:
Your README is lacking a lot of fields, also it seems as if your using it as Documentation instead of something like Swagger. I suggest you read up on what a README should contain and revise this one.